README.md

```diff
@@ -41,6 +41,23 @@ described earlier.
     cargo test
 
 
+Travis
+------
+CSearch is continuously integrated on [Travis CI].
+
+To update encrypted credentials in the `.travis.yml` file, use the
+[Travis command line tool].
+
+    gem install travis
```

Comment:
Alternatively, to stick with our spiel here, there's a docker container so you don't have to deal with terrible third-party package managers: https://hub.docker.com/r/caktux/travis-cli/